imports/decorators/infiniteScroll.js

```diff
@@ -13,6 +14,10 @@ export default (reducer = defaultReducer, options = defaultOptions) => (WrappedC
       done: PropTypes.bool,
     }
 
+    componentWillMount() {
+      mergedOptions = { ...defaultOptions, ...options };
+    }
```

Comment:
@jbaxleyiii Here's the updated code using the spread.  I had thought this might be the preferred method, but wasn't sure.